Doc appts

Thommond opened the pull request:
Created the docappt.php file so supervisor and admin can schedule appointments for different doctors based on date.

Updated styles to tweak minor stylings like success message and display_only class.

Thommond committed:
Created the docappt.php file which supervisor and admin can access to assign doctor appointments. 

Updated style.css for minor style fixs on docappt.php page. 

Updated adminHome.php and supervisorHome.php to have link to docappt.php 
file.

Thommond committed:
Added success message style to style.css

Updated teh docapp.php to echo contents into second form then insert 
data into the appointments table to make a new appointment.

BrendanHorst commented on views/admin/docappt.php:
```diff
@@ -0,0 +1,148 @@
+
+        <label for="patient_id">Patient ID:
+        <input type="text" name="patient_id">
+        </label>
```
this is not indented properly.
Also, putting the input inside of the label is an alternative to using `for`, so you don't need to have both

BrendanHorst commented on views/admin/docappt.php:
```diff
@@ -0,0 +1,148 @@
+        if(mysqli_query($db_link, $sql)) {
+          $result = mysqli_fetch_row(mysqli_query($db_link, $sql));
+
+          $patient_name =  $result[0] . ' ' . $result[1];
```
There should be a check for if the result comes back empty or else this will error.

BrendanHorst commented on views/admin/docappt.php:
```diff
@@ -0,0 +1,148 @@
+          if(mysqli_query($db_link, $sql)) {
+            $result = mysqli_fetch_row(mysqli_query($db_link, $sql));
+            $id = $result[0];
+          }
```
if there is no roster for the selected date the query will work but the result will be empty, which will cause in an error with `result[0]`

BrendanHorst reviewed (changes requested):
Whenever you construct a select query using form input, it would be best to check if the query actually returned something before attempting something like `result[0]`, as that would error if the input was incorrect or the table was empty.

Thommond committed:
Updated two database result querys to have checks if empty.

Updated indentation and label semantics.

Updated where the html is echoed in php from line 87 to line 107
